playbooks/ansible-network-vyos-appliance/files/bootstrap.yaml

```diff
@@ -9,7 +9,8 @@
   tasks:
     - name: lookup SSH public key
       set_fact:
-        _ssh_key: "{{ lookup('file', '~/.ssh/id_rsa.pub') }}"
+        _ssh_file: "~/.ssh/{{ zuul_ssh_key_dest }}.pub"
```

Comment:
zuul variables are out of scope here, i think in this case, it is okay to replace rsa with ecdsa hardcoded